lib/ae_page_objects/elements/collection.rb

```diff
@@ -66,10 +73,9 @@ def item_class_at(index)
 
     def item_xpath
       @item_xpath ||= begin
-        evaled_locator = eval_locator(@item_locator)
-        
-        query_args = evaled_locator + [{:exact => true}]
-        query = Capybara::Query.new(*query_args)
+        evaled_locator = eval_locator(@item_locator).clone
+        evaled_locator.pop if evaled_locator.last.is_a?(::Hash)
+        query = Capybara::Query.new(*evaled_locator, options)
```

Comment:
Also, this logic seems confusing. We're dropping the last element of `evaled_locator` if it's a hash, and then just adding it back on? It seems more straightforward to _add_ a hash if the last element wasn't a hash. That's what the old code was doing, but it wasn't merging the hash if one was already there. Can we just do that instead? What if we override `eval_locator` in this class? Would that make this implementation simpler?